.github/workflows/release.yml

```
@@ -0,0 +1,115 @@
+    cancel-in-progress: true
+
+permissions:
+    contents: read
+
+jobs:
+    build:
+        name: Build, test, and validate
+        runs-on: ubuntu-latest
+        timeout-minutes: 20
+        steps:
+            -   name: Set Git Options
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)